Wrappers/Python/cil/processors/__init__.py

```diff
@@ -1,23 +1,25 @@
-# -*- coding: utf-8 -*-
-#  CCP in Tomographic Imaging (CCPi) Core Imaging Library (CIL).
-
-#   Copyright 2017 UKRI-STFC
-#   Copyright 2017 University of Manchester
-
-#   Licensed under the Apache License, Version 2.0 (the "License");
-#   you may not use this file except in compliance with the License.
-#   You may obtain a copy of the License at
-
-#   http://www.apache.org/licenses/LICENSE-2.0
-
-#   Unless required by applicable law or agreed to in writing, software
-#   distributed under the License is distributed on an "AS IS" BASIS,
-#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#   See the License for the specific language governing permissions and
-#   limitations under the License.
-
-from .CentreOfRotationCorrector import CentreOfRotationCorrector
-from .CofR_xcorr import CofR_xcorr
-from .Normaliser import Normaliser
-from .Resizer import Resizer
-from .RingRemover import RingRemover
+# -*- coding: utf-8 -*-

```

Comment:
Please update the headers as in [here](https://github.com/vais-ral/CCPi-Framework/issues/540#issuecomment-718794402)